task_02/index.html

```diff
@@ -1,11 +1,195 @@
+                sit amet, consectetur, adipisci velit, sed quia non numquam eius
+                modi tempora incidunt ut labore et dolore magnam aliquam quaerat
+                voluptatem. Ut enim ad minima veniam, quis nostrum
+                exercitationem ullam corporis suscipit laboriosam, nisi ut
+                aliquid ex ea commodi consequatur? Quis autem vel eum iure
+                reprehenderit qui in ea voluptate velit esse quam nihil
+                molestiae consequatur, vel illum qui dolorem eum fugiat quo
+                voluptas nulla pariatur?"
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="item" id="item4">
+          <div class="section-wrapper">
+            <div class="text-wrapper">
+              <div class="post-title">This is a title</div>
+              <div class="post-text">
+                "Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+                accusantium doloremque laudantium, totam rem aperiam, eaque ipsa
+                quae ab illo inventore veritatis et quasi architecto beatae
+                vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia
+                voluptas sit aspernatur aut odit aut fugit, sed quia
+                consequuntur magni dolores eos qui ratione voluptatem sequi
+                nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor
+                sit amet, consectetur, adipisci velit, sed quia non numquam eius
+                modi tempora incidunt ut labore et dolore magnam aliquam quaerat
+                voluptatem. Ut enim ad minima veniam, quis nostrum
+                exercitationem ullam corporis suscipit laboriosam, nisi ut
+                aliquid ex ea commodi consequatur? Quis autem vel eum iure
+                reprehenderit qui in ea voluptate velit esse quam nihil
+                molestiae consequatur, vel illum qui dolorem eum fugiat quo
+                voluptas nulla pariatur?"
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </div>
+  </body>
+  <script>
+    function moveText() {
+      if (document.getElementById("toggleCheckbox").checked === true) {
+        console.log(true);
+
+        var section = document.querySelectorAll(".section-wrapper");
+        section.forEach(el => {
+          el.style.position = "absolute";
+          el.style.right = "-250px";
+        });
+
+        var sectionTag = document.getElementById("section");
+        sectionTag.style.position = "fixed";
+      } else {
+        setTimeout(() => {
+          var section = document.querySelectorAll(".section-wrapper");
+          section.forEach(el => {
+            el.style.removeProperty("position");
+            el.style.removeProperty("right");
+          });
+          var sectionTag = document.getElementById("section");
+          sectionTag.style.removeProperty("position");
```

Comment:
Can we move both `sectionTag` and `section` variable at the beginning of `moveText` function, since we are using in both case `if/else`? Also, can we use `const`? Later I will check what this function does in details

Reply:
We can move it. We can use const since we are pointing at the DOM element which is static and we don't need to change it.